pkg/runtime/runtime_test.go

```diff
@@ -1374,11 +1374,6 @@ func TestInitPubSub(t *testing.T) {
 		rt.appChannel = mockAppChannel
 
 		// User App subscribes 2 topics via http app channel
-		fakeReq := invokev1.NewInvokeMethodRequest("dapr/subscribe").
```

Comment:
Reviewers: no idea why changes to these tests were needed. This doesn't really change the test, just makes the matcher a bit more lax:

- Before, it required to receive the exact `fakeReq` object
- Now it requires an object which is a request object for `dapr/subscribe`

The change to the matcher is similar to what's used in this test file in many other places.

Without this change, the matcher doesn't recognize the object because a property inside is `nil` rather than another zero value.

I have no idea why this test worked as-is in master and fails here, since nothing should have changed. But this fixes the test - and it's arguably more robust anyways.

Reply:
From the failing test, where the matcher was failing:

![image](https://user-images.githubusercontent.com/43508/210458838-89b09617-c57e-485c-b185-0f97f36e7322.png)

🤷🤷🤷

Reply:
Actually I think I know now why this is happening... In master, resiliency is enabled in unit tests, but with this PR it is not. (it's still enabled in E2E tests)

Reply:
Changed to add a way to enable features in unit tests